pkg/setting/setting_sentry.go

```diff
@@ -0,0 +1,16 @@
+package setting
+
+type Sentry struct {
+	Enabled        bool   `json:"enabled"`
+	Dsn            string `json:"dsn"`
```

Comment:
[Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case.](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)
```suggestion
	DSN            string `json:"dsn"`
```